pkg/networkpolicy/controller.go

```diff
@@ -67,6 +68,24 @@ type Config struct {
 	QueueID                    int
 	NodeName                   string
 	NetfilterBug1766Fix        bool
+	NFTableName                string // if other projects use this controllers they need to be able to use their own table name
+}
+
+func (c *Config) Defaults() error {
+	var err error
+	if c.QueueID == 0 {
+		c.QueueID = 100
+	}
+	if c.NodeName == "" {
+		c.NodeName, err = os.Hostname()
+		if err != nil {
+			return err
+		}
+	}
+	if c.NFTableName == "" {
```

Comment:
I was about to ask for "do you have docs for config options" but then I saw the nice:
```
	flag.IntVar(&queueID, "nfqueue-id", 100, "Number of the nfqueue used")
```
flags one can pass, should we add the new tableName to that? or is there a difference between internal and external configs?

Reply:
yeah, I don't want to expose the table name to the users of kube-network-policies , though I want to expose it to the consumer of the controller, in this case kindnet